### ISSUE TITLE
Remove TSLint

### ISSUE THREAD

**Issue by rhlsthrm** [ConnextProject/indra-v2#123]
https://github.com/typescript-eslint/typescript-eslint#what-about-tslint

**Comment by hthillman** [comment]
Can this be closed?

**Comment by rhlsthrm** [comment]
No, this is still an issue. We should do this, since TSLint is being deprecated.
